src/OpenTelemetry/Metrics/Aggregators/Aggregator.cs

```diff
@@ -18,15 +18,34 @@
 
 namespace OpenTelemetry.Metrics.Aggregators
 {
+    /// <summary>
+    /// Aggregator base class.
+    /// </summary>
+    /// <typeparam name="T">The type.</typeparam>
     public abstract class Aggregator<T>
         where T : struct
     {
+        /// <summary>
+        /// Adds value to the running total in a thread safe manner.
+        /// </summary>
+        /// <param name="value">Value to be added to running total.</param>
```

Comment:
"Value to be aggregated." - this should suffice. Mention total/add etc assumes too much about the implementation.